pkg/services/featuremgmt/registry.go

```diff
@@ -816,5 +816,12 @@ var (
 			FrontendOnly: false,
 			Owner:        grafanaAlertingSquad,
 		},
+		{
+			Name:         "enableNativeHTTPHistogram",
```

Comment:
Given that there will be (hopefully, eventually?) more than a single native histogram, IMO this should be plural. Also I think we should omit `HTTP` - there's no reason to _only_ expose native histograms for HTTP metrics under this flag:

```suggestion
			Name:         "enableNativeHistograms",
```

Reply:
I only thought about the HTTP histogram but you are right that we have more histograms to convert. But perhaps we more toggles? Once we a closer to replacing all histograms we could have a dedicated toggle for that.